cs/v1/CS_v1.0.0.yaml

```diff
@@ -228,7 +242,15 @@ paths:
           name: cursor
           description: 'A server generated value to specify a specific point in a collection result, used for pagination.'
         - $ref: '#/components/parameters/Api-Version-Major'
-      description: Provides the product offering of single or multiple estimated end-to-end route options for a shipment in the pre-booking phase.
+      description: |
+        Provides the product offering of single or multiple estimated end-to-end route options for a shipment in the pre-booking phase. This includes point-to-point specification of all transport legs, estimated timings, estimated schedules and interdependencies between transport legs.
+
+        The list of solutions returned in the response can be tailored to a specific need by combining available query parameters.
+
+        The minimum required query parameters are `placeOfReceipt` and `placeOfDelivery`. If no further query parameters are used to tailor the response, the provider of the GET endpoint will return their best suggestions in the response. 
+
+        The `GET /v1/point-to-point-routes` endpoint can be implemented independently of having implemented the `GET /v1/port-schedules`  and `GET /v1/vessel-schedules`  endpoints.
```

Comment:
I have other places used the code-markdown when listing endPoints. I think it looks better

Reply:
<img width="374" alt="image" src="https://github.com/dcsaorg/DCSA-OpenAPI/assets/3489388/8fce2f74-4e27-420c-98e9-25d052d21e91">